pkgs/graphs/CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+Want to contribute? Great! First, read this page (including the small print at
```

Comment:
Not necessary for this PR (can be done after its merged in), but we can delete this separate contributing file.